datasets/ladi.yaml

```diff
@@ -0,0 +1,56 @@
+Name: Low Altitude Disaster Imagery (LADI) Dataset
+Description: The Low Altitude Disaster Imagery (LADI) Dataset consists of human and machine annotated airborne images collected by the Civil Air Patrol in support of various disaster responses from 2015-2019. The initial release of LADI focuses on the Atlantic hurricane seasons and coastal states along the Atlantic Ocean and Gulf of Mexico. Annotations are included for major hurricanes of Harvey, Maria, and Florence. Two key distinctions are the low altitude, oblique perspective of the imagery and disaster-related features, which are rarely featured in computer vision benchmarks and datasets.
+Documentation: https://github.com/LADI-Dataset/ladi-overview
+Contact:  ladi-dataset-admin@mit.edu
+ManagedBy: "[MIT Beaver Works](https://beaverworks.ll.mit.edu/)"
+UpdateFrequency: Periodically
+License: Creative Commons Attribution 4.0 International (CC BY 4.0) 
+Tags:
+  - aws-pds
+  - aerial imagery
+  - coastal
+  - computer vision
+  - disaster response
+  - earth observation
+  - earthquakes
+  - geospatial
+  - imaging
+  - image processing
+  - infrastructure
+  - land
+  - machine learning
+  - mapping
+  - natural resource
+  - seismology
+  - transportation
+  - urban
+  - water
+Resources:
+  - Description: "LADI dataset: images and labels"
+    ARN: arn:aws:s3:::ladi
+    Region: us-west-2
+    Type: S3 Bucket
+DataAtWork:
+  Tutorials:
+    - Title: LADI Tutorials
+      URL: https://github.com/LADI-Dataset/ladi-tutorial
+      AuthorName: Andrew Weinert, Jianyu Mao, Kiana Harris, Nae-Rong Chang, Caleb Pennell, Yiming Ren, Ryan Earley, Nadia Dimitrova
+      AuthorURL: https://github.com/LADI-Dataset
+  Tools & Applications:
+    - Title: NIST TRECVID 2020 - Disaster Scene Description and Indexing (DSDI)
+      URL: https://www-nlpir.nist.gov/projects/tv2020/dsdi.html
+      AuthorName: TREC Video Retrieval Evaluation (TRECVID)
+    - Title: Remote Sensing for Disaster Response Course
+      URL: https://github.com/bwsi-hadr
+      AuthorName: Beaver Works Summer Institute
+      AuthorURL: https://beaverworks.ll.mit.edu/CMS/bw/bwsi
+  Publications:
+   - Title: Large Scale Organization and Inference of an Imagery Dataset for Public Safety
+      URL: https://doi.org/10.1109/HPEC.2019.8916437
```

Comment:
Indentation is off here, need to remove a space

Reply:
Addressed in fbe186b